.github/workflows/ci-build.yml

```diff
@@ -1,4 +1,4 @@
-name: Voron CI
+name: Voron Tap CI
```

Comment:
Please revert the name change here, this job is not specific to Voron Tap, it's the same job as the other repositories.